extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -313,6 +314,15 @@ private OpenAPI generateAnnotationModel(IndexView indexView, Capabilities capabi
 
         String defaultPath = config.getValue("quarkus.http.root-path", String.class);
 
+        Optional<String> resteasyPath = config.getOptionalValue("quarkus.resteasy.path", String.class);
```

Comment:
Why are we not using https://github.com/quarkusio/quarkus/blob/master/extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java#L143 here?

Reply:
Hi @machi1990 - we do not want a dependency on RestEasy. See https://github.com/quarkusio/quarkus/issues/8407

Reply:
I see, thanks for the linked issue. I guess moving the config to an `spi` module could have been a solution. 

We certainly should not be doing that in this PR but using the quarkus config object is more reliable than manually retrieving them from MP Config.

Reply:
Cool. Thanks for the insights. Agree, let's do this in another PR